duckduckgo/iOS
# Smartling translation

Task/Issue URL: https://app.asana.com/0/0/1202854620659929/f

**Description**:
Add translations for accessibility improvements 

**Steps to test this PR**:
1. Open the app on another supported language, check if everything works fine
2. Check if voice-over is speaking on the correct language

**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
